README.md

```diff
@@ -857,9 +857,11 @@ See also [View Settings](#view-settings- 'Jump to the View settings')
 
 ## Menu & Toolbar Settings [#](#menu--toolbar-settings- 'Menu & Toolbar Settings')
 
-| Name            | Description                                           |
-| --------------- | ----------------------------------------------------- |
-| `gitlens.menus` | Specifies which commands will be added to which menus |
+| Name                                   | Description                                                                                                          |
+| -------------------------------------- | -------------------------------------------------------------------------------------------------------------------- |
+| `gitlens.menus`                        | Specifies which commands will be added to which menus                                                                |
+| `gitlens.menus.annotations.behaviour`  | Specifies whether the file annotations button in the editor title acts as a menu (default) or a toggle button        |
```

Comment:
Just need to remove this and document the new `gitlens.fileAnnotations.command`

Reply:
I've made the changes